src/include/cpu/power/scom_registers.h

```diff
@@ -0,0 +1,30 @@
+typedef enum
+{
+    PB_WEST_MODE_CFG =              0x501180A, // Power Bus PB West Mode Configuration Register
```

Comment:
@IgorBagnucki Let's use defines instead of enum

Reply:
I have changed it to #define's